pkg/clusteragent/clusterchecks/dispatcher_configs.go

```diff
@@ -10,7 +10,7 @@ package clusterchecks
 import (
 	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
 	"github.com/DataDog/datadog-agent/pkg/clusteragent/clusterchecks/types"
-	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	"github.com/DataDog/datadog-agent/pkg/collector/check/id"
```

Comment:
could you define an import name for this package like `check_id` because `id` is to short to give any context in the code after

Reply:
Fixed by: https://github.com/DataDog/datadog-agent/pull/17929/commits/a86c685ed917bf8eb9c892e7eb17d9e09aa1f54a